pinot-segment-local/src/main/java/org/apache/pinot/segment/local/segment/creator/impl/SegmentColumnarIndexCreator.java

```diff
@@ -211,7 +208,15 @@ public void init(SegmentGeneratorConfig segmentCreationSpec, SegmentIndexCreatio
       }
 
       if (textIndexColumns.contains(columnName)) {
-        _textIndexCreatorMap.put(columnName, _indexCreatorProvider.newTextIndexCreator(context.forTextIndex(true)));
+        Map<String, String> columnProperties = _columnProperties.get(columnName);
+        FSTType fstType = FSTType.LUCENE;
+
```

Comment:
(minor, readability) Can we remove the empty lines in this if block? I find them a little bit hard to read (the logic flow for the empty line separated blocks are continuous)